include/rewd_controllers/MoveUntilTouchController.hpp

```diff
@@ -6,7 +6,7 @@
 #include <hardware_interface/force_torque_sensor_interface.h>
 #include <hardware_interface/joint_command_interface.h>
 #include <pr_control_msgs/SetForceTorqueThresholdAction.h>
-#include <pr_hardware_interfaces/TriggerableInterface.h>
+#include <rewd_controllers/TriggerableInterface.hpp>
```

Comment:
In lines 24 and 76, there are two references to the pr_hardware_interfaces namespace left.

Reply:
> not sure how to leave a comment on lines that are unchanged

In "File changed" tap, if you put your mouse cursor on slightly right to the line numbers, a blue button with plus icon will appear. You can leave a comment on lines by clicking the blue button.

Reply:
Thanks, but if I press the button on the bottom to show lines that are unchanged, I can hover over these lines, but the blue button doesn't appear for some reason...

Reply:
Yeah, it seems GitHub only allows to add a comment for changed lines. :disappointed: